parts/linux/cloud-init/artifacts/aks-log-collector.sh

```diff
@@ -99,62 +99,6 @@ command -v crictl >/dev/null && {
   crictl statsp -o json >collect/crictl_statsp.json 2>&1
 }
 
-# Collect network information
-command -v conntrack >/dev/null && {
-  conntrack -L >collect/conntrack.txt 2>&1
-  conntrack -S >>collect/conntrack.txt 2>&1
-}
-command -v ip >/dev/null && {
-  ip -4 -d -j addr show >collect/ip_4_addr.json 2>&1
-  ip -4 -d -j neighbor show >collect/ip_4_neighbor.json 2>&1
-  ip -4 -d -j route show >collect/ip_4_route.json 2>&1
-  ip -4 -d -j tcpmetrics show >collect/ip_4_tcpmetrics.json 2>&1
-  ip -6 -d -j addr show table all >collect/ip_6_addr.json 2>&1
-  ip -6 -d -j neighbor show >collect/ip_6_neighbor.json 2>&1
-  ip -6 -d -j route show table all >collect/ip_6_route.json 2>&1
-  ip -6 -d -j tcpmetrics show >collect/ip_6_tcpmetrics.json 2>&1
-  ip -d -j link show >collect/ip_link.json 2>&1
-  ip -d -j netconf show >collect/ip_netconf.json 2>&1
-  ip -d -j netns show >collect/ip_netns.json 2>&1
-  ip -d -j rule show >collect/ip_rule.json 2>&1
-}
-command -v iptables >/dev/null && iptables -L -vn --line-numbers >collect/iptables.txt 2>&1
-command -v ip6tables >/dev/null && ip6tables -L -vn --line-numbers >collect/ip6tables.txt 2>&1
-command -v nft >/dev/null && nft -jn list ruleset >collect/nftables.json 2>&1
-command -v ss >/dev/null && {
-  ss -anoempiO --cgroup >collect/ss.txt 2>&1
-  ss -s >>collect/ss.txt 2>&1
-}
-
-# Collect network information from network namespaces
```

Comment:
I only think you need to remove the ip_netns_commands collection, just the stuff using exec

Reply:
discussed offline, the other bits are being removed due to OOM issue observed in customer environments as a result of the `nft` calls